<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Alicia Pliego" />


<title>RNAseq NTRK</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/sandstone.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="site_libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.5.1/plotly-latest.min.js"></script>
<link href="site_libs/vega-1/vega_plots.css" rel="stylesheet" />
<script src="site_libs/vega-1/vega.min.js"></script>
<link href="site_libs/vega-tooltip-1/vega-tooltip.css" rel="stylesheet" />
<script src="site_libs/vega-tooltip-1/vega-tooltip.js"></script>
<script src="site_libs/GlimmaV2-1/MDSSpecs.js"></script>
<script src="site_libs/GlimmaV2-1/XYSpecs.js"></script>
<script src="site_libs/GlimmaV2-1/expressionSpec.js"></script>
<script src="site_libs/GlimmaV2-1/makeTooltip.js"></script>
<script src="site_libs/GlimmaV2-1/saveTools.js"></script>
<script src="site_libs/glimmaMDS-binding-2.0.0/glimmaMDS.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">RNAseq-NTRK</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="de_analysis.html">Clustering analysis</a>
</li>
<li>
  <a href="differential_expression_NTRK.html">Differential expression</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">RNAseq NTRK</h1>
<h4 class="author">Alicia Pliego</h4>
<h4 class="date">3/11/2022</h4>

</div>


<div id="rnaseq-ntrk-analysis" class="section level1" number="1">
<h1 number="1"><span class="header-section-number">1</span> RNAseq NTRK analysis</h1>
<div id="research-questions" class="section level2" number="1.1">
<h2 number="1.1"><span class="header-section-number">1.1</span> Research questions</h2>
<div id="htmlwidget-f3c3d131d03bc6469c51" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-f3c3d131d03bc6469c51">{"x":{"visdat":{"454d18dc85d3":["function () ","plotlyVisDat"]},"cur_data":"454d18dc85d3","attrs":{"454d18dc85d3":{"header":{"values":["<b>Samples<\/b>","Sample","Annotation"],"align":["left","center","center"],"line":{"width":1,"color":"black"},"fill":{"color":"rgb(235, 100, 230)"},"font":{"family":"Arial","size":14,"color":"white"}},"cells":{"values":[["1","2","3","4","5","6","7","8","9","10","11"],["LR2022_14","LR2022_15","LR2022_16","LR2022_17","LR2022_18","LR2022_19","LR2022_20","LR2022_21","LR2022_22","LR2022_23","LR2022_24"],["bNaive","bNaive","bNaive","bNaive","bNaive","aResistant","aResistant","aResistant","aResistant","aResistant","aResistant"]],"align":["left","center","center"],"line":{"color":"black","width":1},"fill":{"color":["rgb(235, 193, 238)","rgba(228, 222, 249, 0.65)"]},"font":{"family":"Arial","size":12,"color":"black"}},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"table"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"header":{"values":["<b>Samples<\/b>","Sample","Annotation"],"align":["left","center","center"],"line":{"width":1,"color":"black"},"fill":{"color":"rgb(235, 100, 230)"},"font":{"family":"Arial","size":14,"color":"white"}},"cells":{"values":[["1","2","3","4","5","6","7","8","9","10","11"],["LR2022_14","LR2022_15","LR2022_16","LR2022_17","LR2022_18","LR2022_19","LR2022_20","LR2022_21","LR2022_22","LR2022_23","LR2022_24"],["bNaive","bNaive","bNaive","bNaive","bNaive","aResistant","aResistant","aResistant","aResistant","aResistant","aResistant"]],"align":["left","center","center"],"line":{"color":"black","width":1},"fill":{"color":["rgb(235, 193, 238)","rgba(228, 222, 249, 0.65)"]},"font":{"family":"Arial","size":12,"color":"black"}},"type":"table","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<div id="visualizing-library-sizes" class="section level3" number="1.1.1">
<h3 number="1.1.1"><span class="header-section-number">1.1.1</span> Visualizing library sizes</h3>
<p>Sample 32 and sample 26 have low read number.</p>
<div id="htmlwidget-4f62da93c5248ec9aeb7" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4f62da93c5248ec9aeb7">{"x":{"visdat":{"454d6209ef73":["function () ","plotlyVisDat"]},"cur_data":"454d6209ef73","attrs":{"454d6209ef73":{"x":["LR2022_14","LR2022_15","LR2022_16","LR2022_17","LR2022_18","LR2022_19","LR2022_20","LR2022_21","LR2022_22","LR2022_23","LR2022_24"],"y":[26528657,33060587,33676677,35578680,32204739,29896944,27320278,19139199,26493172,38055200,31416000],"name":"Barplot of library sizes","alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"bar"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"shapes":[{"type":"line","x0":0,"x1":1,"xref":"paper","y0":30000000,"y1":30000000,"line":{"color":"red"}},{"type":"rect","line":{"color":"black"},"x0":0.9,"x1":2}],"plot_bgcolor":"#e5ecf6","title":"Barplot of library sizes","xaxis":{"domain":[0,1],"automargin":true,"title":"Samples","type":"category","categoryorder":"array","categoryarray":["LR2022_14","LR2022_15","LR2022_16","LR2022_17","LR2022_18","LR2022_19","LR2022_20","LR2022_21","LR2022_22","LR2022_23","LR2022_24"]},"yaxis":{"domain":[0,1],"automargin":true,"title":"Million of reads"},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"x":["LR2022_14","LR2022_15","LR2022_16","LR2022_17","LR2022_18","LR2022_19","LR2022_20","LR2022_21","LR2022_22","LR2022_23","LR2022_24"],"y":[26528657,33060587,33676677,35578680,32204739,29896944,27320278,19139199,26493172,38055200,31416000],"name":"Barplot of library sizes","type":"bar","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="filtering-non-expressed-genes" class="section level3" number="1.1.2">
<h3 number="1.1.2"><span class="header-section-number">1.1.2</span> Filtering non-expressed genes</h3>
<ul>
<li>Remove genes with a count lower than 5 Plot the number of samples a gene is expressed in: <img src="de_analysis_files/figure-html/unnamed-chunk-8-1.png" width="672" style="display: block; margin: auto;" /></li>
</ul>
<p>It seems that genes are either expressed in all samples, or not expressed at all. Only keep genes that are expressed in at least 2 samples.</p>
<ul>
<li>Filter genes that are expressed in at least 2 samples.</li>
</ul>
</div>
<div id="visualize-count-distributions" class="section level3" number="1.1.3">
<h3 number="1.1.3"><span class="header-section-number">1.1.3</span> Visualize count distributions</h3>
<p>Applying the boxplot function to the raw counts from our dataset reveals something about the nature of the data; the distributions are dominated by a few genes with very large counts. <img src="de_analysis_files/figure-html/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Log2 scale data with the vsd function from DEseq2 to remove the dependence of the variance on the mean, particurarly the high variance of the logarithm of count data when the mean is low.</p>
</div>
<div id="mds-plot" class="section level3" number="1.1.4">
<h3 number="1.1.4"><span class="header-section-number">1.1.4</span> MDS plot</h3>
<p>The multidimensional scaling (MDS) plot is frequently used to explore differences in samples. When data has been MDS transformed, the first two dimensions explain the greatest variance between samples, and the amount of variance decreases monotonically with increasing dimension.</p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="heatmap-of-the-sample-to-sample-distances" class="section level3" number="1.1.5">
<h3 number="1.1.5"><span class="header-section-number">1.1.5</span> Heatmap of the sample-to-sample distances</h3>
<p>I applied the dist function to the transpose of the transformed count matrix to get sample-to-sample distances.</p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="dispersion-plot" class="section level3" number="1.1.6">
<h3 number="1.1.6"><span class="header-section-number">1.1.6</span> Dispersion plot</h3>
<p>The dispersion plot below is typical, with the final estimates shrunk from the gene-wise estimates towards the fitted estimates. Some gene-wise estimates are flagged as outliers and not shrunk towards the fitted value, (this outlier detection is described in the manual page for estimateDispersionsMAP). The amount of shrinkage can be more or less than seen here, depending on the sample size, the number of coefficients, the row mean and the variability of the gene-wise estimates. <img src="de_analysis_files/figure-html/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>par(mfrow=c(2,2),mar=c(2,2,1,1))
ylim &lt;- c(-10.5,10.5)
resGA &lt;- results(dds, lfcThreshold=2.5, altHypothesis=&quot;greaterAbs&quot;)
resLA &lt;- results(dds, lfcThreshold=2.5, altHypothesis=&quot;lessAbs&quot;)
resG &lt;- results(dds, lfcThreshold=2.5, altHypothesis=&quot;greater&quot;)
resL &lt;- results(dds, lfcThreshold=2.5, altHypothesis=&quot;less&quot;)
drawLines &lt;- function() abline(h=c(-2.5,2.5),col=&quot;dodgerblue&quot;,lwd=2)
plotMA(resGA, ylim=ylim); drawLines()
plotMA(resLA, ylim=ylim); drawLines()
plotMA(resG, ylim=ylim); drawLines()
plotMA(resL, ylim=ylim); drawLines()</code></pre>
<p><img src="de_analysis_files/figure-html/unnamed-chunk-20-1.png" width="672" /></p>
</div>
</div>
</div>


<p>Copyright &copy; 2022 Alicia Pliego. FTP lab. All rights reserved.</p>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
